<?xml version="1.0"?>

<launch>
  
  
  <node name="grid_map_2d_mapper" type="grid_map_2d_mapper_node" pkg="grid_map_2d_mapper" output="screen" respawn="true">
      <param name="min_height" value="0.25"/>    
      <param name="max_height" value="0.8"/>
      <param name="range_min" value="0.2"/>
      <param name="target_frame" value="base_stabilized"/>      
      <param name="downsample_cloud" value="false"/>     
      <param name="probability_free" value="0.5"/>
      <param name="probability_occupied" value="0.8"/> 
  </node>
  

</launch>
